### DISCUSSION TITLE
Can we deprecate this package?

### DISCUSSION THREAD

**Discussion by chalkygames123** [Mister-Hope/gulp5-sass-plugin#119] [General]
Hi,

Yesterday, gulp-sass v5 which drops dependency on node-sass has been finally released. I'm wondering if we can deprecate this package now, is there any reason to continue using it?

https://github.com/dlmanning/gulp-sass/releases/tag/v5.0.0

**Comment by Mister-Hope** [comment]
I won't deprecate this package because this package:

- are exporting functions through `exports` which has backword compability with esm syntax (while `gulp-sass@v5` is not beacuse of still exporing a function directly thourhg `module.exports`)
- provides type decalration files for typescript projects

and

- is based on new tools which is easier to keep maintaining
- has a coverage test and some updated test comparing to the old repo


The first one is readlly important for me because my gulp config file are generated thourgh ts files. And I am using esm syntax in ts.

**Comment by chalkygames123** [comment]
Makes sense. Thank you for the explanation! I hope gulp-sass could catch up with those advantages someday.
